cmake/EmbedLLVM.cmake

```diff
@@ -0,0 +1,40 @@
+enable_language(ASM)
+
+function(EMBED_LLVM OUTPUT_FILE)
```

Comment:
IMHO this function really needs some short and precise doc. I read over it and I can but guess what it does. The embedded assembly is probably beyond anyone but you and could use a rough, doesn't need to be in depth, comment as well.

Reply:
Questions I'd probably like answered are: What is a CPP_BUNDLE_FILE, a LLVM_BUNDLE_FILE, why are we invoking clang here in the first place, how does this fit into the architecture of how the JIT is embedded into Hyrise?